static/lang/re-en.json

```diff
@@ -171,7 +173,12 @@
                 "SunNote": "The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} for 1 round."
             },
             "DaemonForm": {
-                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage"
+                "AbrikandiluJawsCriticalHitNote": "On a critical hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.i3OJZU2nk64Df3xm]{Clumsy 1} for 1 minute.",
+                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage",
+                "KithangianPincerGrabNote": "You can spend an action after a hit to attempt to @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Tkd8sH4pwFIPzqTr]{Grab} the target."
+            },
+            "DevilForm": {
+                "LevalochSavesAgainstMoveAllLabel": "Resist being moved against your will" 
```

Comment:
Maybe move this to `PF2E.GenericLabel.Resist.ForcedMovement` and just make it `vs. forced movement`